kedro-datasets/kedro_datasets/constants.py

```diff
@@ -0,0 +1,11 @@
+from typing import NewType
```

Comment:
I think it will be helpful to have a short explanation at the top of the file of what these constants are for.

Reply:
Agree. Is there going to an example in the documentation to show that you can extend custom dataset for this preview function?